datalad/support/tests/test_repo_save.py

```diff
@@ -15,6 +15,7 @@
     with_tempfile,
     eq_,
     known_failure_githubci_win,
```

Comment:
`known_failure_githubci_win` is no longer used in this file (perhaps worth fixing up locally right after merge)